cves/CVE-2012-0022.yml

```diff
@@ -65,11 +67,11 @@ fixes:
      note: SVN rev 1224640, from the Tomcat website.
    - commit: b05497eff4311a9657de6dfc53511d0309eb9db4
      note: SVN rev 1228191, from the Tomcat website.
-   - commit:
-     note:
+   - commit: 24272405fd444467421d4919ca0aed55b906c83b
```

Comment:
Could you please write a summary of how the commit introduced the issue? I would like to know a bit more about the sort of design flaws that cause an error due to unbounded parameters. I have trouble picturing how such erroneous code/input works.